oak-api/src/main/java/org/apache/jackrabbit/oak/api/jmx/IndexStatsMBean.java

```diff
@@ -127,6 +128,15 @@ public interface IndexStatsMBean {
      */
     String getReferenceCheckpoint();
 
+    @Description("Force update the indexing lane to a latest checkpoint. This will abort and pause the running lane, release it's lease and set the reference checkpoint to a latest one." +
```

Comment:
typo
```suggestion
    @Description("Force update the indexing lane to a latest checkpoint. This will abort and pause the running lane, release its lease and set the reference checkpoint to a latest one." +
```